### DISCUSSION TITLE
第5节课笔记(7班)

### DISCUSSION THREAD

**Discussion by JimmyMa99** [InternLM/Tutorial#336] [笔记]
第5节课笔记(7班)

**Comment by createup2023** [comment]
https://www.yuque.com/creatup/ynrsde/tvexmdz0zm0u97hd?singleDoc# 《第五次笔记：》
补充CSDN笔记：https://blog.csdn.net/weixin_48339493/article/details/135756520

**Comment by wuhaostudio** [comment]
https://blog.csdn.net/2301_77286822/article/details/135593435?spm=1001.2014.3001.5501

**Comment by SaaRaaS-1300** [comment]
第五节课作业同笔记，github链接：https://github.com/SaaRaaS-1300/InternLM_openNotebook/blob/main/Lesson-5/Lesson-5-Notebook.md

**Comment by anneeeee** [comment]
https://mp.weixin.qq.com/s/ijdlN_4V8J_gYIm-DUgXMQ

**Comment by lifelinezhang** [comment]
https://zhuanlan.zhihu.com/p/678693578

**Comment by PommesPeter** [comment]
https://memo.sylin.host/docs/DeepLearning/%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86/%E8%AF%BE%E7%A8%8B%E5%AD%A6%E4%B9%A0/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/%E7%AC%AC%E4%BA%94%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0

**Comment by hui1feng** [comment]
第五节课笔记：https://github.com/hui1feng/LMDeploy/blob/main/README.md

**Comment by abelezm** [comment]
第五节课作业同笔记，https://blog.csdn.net/weixin_38043453/article/details/135612915

**Comment by leochiang** [comment]
第五次课笔记 https://vvgyb242x83.feishu.cn/docx/HMf8dwjwtoAQHgxlr7dcZuSYncf?from=from_copylink

**Comment by Simon-lyz** [comment]
## InternLM Lesson-5 量化部署
https://g4w90egc30.feishu.cn/docx/S7lPdG6F5ojQcxxFhQ1cYSC6nnh?from=from_copylink

**Comment by Rogue-Trader-zzy** [comment]
笔记：https://www.yuque.com/roguetrader/iu9odz/cao7d50wc0l9vgbd?singleDoc# 《LMDeploy 的量化和部署》
